### ISSUE TITLE
Need more examples

### ISSUE THREAD

**Issue by valyard** [richardszalay/raix#1]
subj.

**Comment by richardszalay** [comment]
Noted and working on it

**Comment by richardszalay** [comment]
Have you checked out the samples in the test bed?http://richardszalay.github.com/rxas/RxAsTestBed/index.htm

**Comment by karfau** [comment]
I stumbled upon this project yesterday. After reading the Overview and all the other "small"/"general" wiki pages, I thought that I had like an overall picture of what was going on.

So I went to Operators and was blazed away of all the options, as I didn't knew how to start nor did understand anything. :(

The resource that helped me most was the [channel9 for the original Rx](http://channel9.msdn.com/Tags/rx) where there is a series that starts with "Reactive Extensions API in depth: ..." 
(15 parts from 1:45 to 5:30 each, about 48 minutes in total)
They really gave me a great insight and understanding about what they are capable of.
Be sure to not miss the download-link if you don't like watching them in the browser or don't have Silverlight installed, it's on the right side of each video.

Then I reread the Operators and could read most of the short descriptions without feeling dumb.
So I'm really looking forward to get my hands on this, to figure out the rest of it.

hope this helps,

karfau
